src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -627,6 +618,7 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
     remoteInfos: VideoStreamDescription[]
   ): void {
     if (!this.tileController) {
+      this.logger.info('tileController not found!');
```

Comment:
Nit:  Feel like this should be error level

Reply:
em...we tend to only use error level for uncontrollable environment-type event, maybe keep it as warn ?